toolkit.lisp

```diff
@@ -13,6 +13,14 @@
 (defun unix->universal (unix)
   (+ unix *unix-epoch-difference*))
 
+(defun universal->utc-timestring (universal)
+  (multiple-value-bind (seconds minutes hours day month year x)
+      (decode-universal-time universal 0)
+    (declare (ignore x))
```

Comment:
Instead of ignoring `x`, simply drop it from the m-v-b bindings.

Reply:
Oh sorry, i forgot to remove it... :(